install/source/chapter07.rst

```diff
@@ -2,46 +2,36 @@
 7 Setup Worker Node for SSL
 #############################
 
-Login to the Web Portal as the master administrator and go to the "Cluster Manager" page. 
+Login to the Web Portal as the master administrator, then, under the menu next to dashboard, go to "Cluster Control Panel".
 
-.. image:: _static/image_s7_1_1.png
+.. image:: _static/image_s7_1_1_v2.png
 
-Click on the "Cluster Server Farm" page to see the "Cluster Worker Nodes".
+After this, go to "Worker Nodes".
 
-.. image:: _static/image_s7_1_2.png
+.. image:: _static/image_s7_1_2_v2.png
 
 There is an "Advanced Setting" icon on the page.
 
-.. image:: _static/image_s7_1_3.png
+.. image:: _static/image_s7_1_3_v2.png
 
 Now, check **"Always force SSL on Login"** and **"Always force SSL for Native Clients"** boxes.
 
 .. warning::
 
   If you use the self-signed SSL certificate, the web portal is the only client that allows you to login after some SSL certificate warning. All other native clients such as Windows, mobile, and Mac clients will reject the connection.
 
-.. image:: _static/image_s7_1_4.png
+.. image:: _static/image_s7_1_4_v2.png
 
 You will also modify the node property for the node.
 
 The External URL will need to match the node's DNS name and the HTTP or HTTPS protocol.
 
-.. image:: _static/image_s7_1_5.png
+.. image:: _static/image_s7_1_5_v2.png
 
 The Node Name needs to match the hostname of the node.
 
-.. image:: _static/image036.jpg
+.. image:: _static/image_s7_1_6_v2.png
```

Comment:
Change the External URL, like in the original image, so users can see that External URL can be different and should be like the official URL for all users to access

Reply:
External URL is a web portal address, in the old image they use https://file.tsys.gladinet.com, you are using an email ABC@xyz.com, should be an URL, you can use the same old one or something like https://acme.example.com